bookdown/03-optimization.Rmd

```diff
@@ -49,7 +49,7 @@ Hyperparameter tuning is supported via the extension package `r mlr_pkg("mlr3tun
 Below you can find an illustration of the process:
 
 ```{r 03-optimization-001, echo = FALSE}
-knitr::include_graphics("images/tuning_process.pdf")
+knitr::include_graphics("images/tuning_process.png")
```

Comment:
But we have a SVG version of this file so this should be included. Additionally the PDF file has to remain in the repository.

Reply:
The problem here is that the graphic does not function properly as svg. The output is somewhat distorted in the top right box as seen here: https://mlr3book.mlr-org.com/tuning.html. I suppose for this graphic it may make more sense using the png.

Reply:
No. It should be possible to generate a SVG that works.

Reply:
I fixed it in the `master`

Reply:
> I fixed it in the master

This means that you have to merge the master branch, Daniel :)

Reply:
@pat-s  thank you! :) 
@jakob-r  I still need to add the separate pdf for the svg files and remove the pdf files from the rmd to resolve the conflict. I was not aware that you cannot do that. I simply assumed the order svg>pdf>png holds. Nor was I aware of the fact that you need to supply both the svg and the respective pdf. Thank you for clarifying this.   
I will come back to this  as soon as possible but most likely on Thursday after my exam.

Reply:
Yes. Maybe I can even improve the README because it is basically quite simple: You have two options:
1) SVG + PDF File but refer to the svg in the Rmd
2) PNG
Any deviation should be avoided.
But this has become clear now? Could have been communicated more clrear from our side as well I guess.

Reply:
Yeah, I think that would be a great improvement! It would really nail down what is actually needed without leaving much room for misunderstanding. Fortunately this has become clear now.